frontend/src/components/static_pages/about.css

```diff
@@ -46,9 +62,15 @@
     }
   }
 
+  #kim-tablet {
```

Comment:
I dont think we need to create and use this id. There is a slightly simpler way.

We can just do this using `kim-image` 

if we change its `margin-top: 20px` inside of this media query -> it should solve the issue! 

There is also a lot of space in-between Our People and Kim's image, and this solution will fix that as well 👍